ciao_contrib/_tools/specextract.py

```diff
@@ -1507,7 +1508,7 @@ def obs_args(self,src_stk,out_stk,bkg_stk,asp_stk,bpixfile_stk,rmffile,
                             v1("Found a Level=3 ahst3.fits file in 'asp' input; the 'asphist' block corresponding to the source region location will be used.\n")
 
                         if stk_count["asol"] != 1:
-                            asphist_arg = f"{stk_dict['asol'][i]}[asphist{self._event_stats(fullfile,'ccd_id')}]"
+                            asphist_arg = f"{asp_stk[i]}[asphist{self._event_stats(fullfile,'ccd_id')}]"
```

Comment:
So is this change a bug fix?

Reply:
yes; weirdly, it's never been triggered in regression testing or anyone reporting this and was only caught by pylint.  Likely wasn't caught since I've yet to see anyone use aspect histograms as input except for our tests....